project/ZipkinResolver.scala

```diff
@@ -49,7 +49,13 @@ object ZipkinResolver extends Plugin {
       }) ++ Seq(
         // the local repo has to be in here twice, because sbt won't push to a "file:"
         // repo, but it won't read artifacts from a "Resolver.file" repo. (head -> desk)
-        "local-lookup" at ("file:" + localRepo.getAbsolutePath),
+        "local-lookup" at {
+          val original = "file:" + localRepo.getAbsolutePath
+          val crossPlatform = localRepo.getAbsoluteFile.toURI.toString
+          println("The original statement says that the repository url is: "+original)
```

Comment:
No, they were just for testing so one of you guys (or Travis) could verify it was correct without starting sbt in debug mode and step through it until you find the value. According to Travis, it looks correct, I'll remove the println statements and push a new commit to this PR.